great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -152,9 +152,9 @@ def add_excel_asset(
         """
         asset = ExcelAsset(
             name=name,
-            base_directory=base_directory,  # type: ignore[arg-type]  # str will be coerced to Path
-            regex=regex,  # type: ignore[arg-type]  # str with will coerced to Pattern
-            order_by=order_by or [],  # type: ignore[arg-type]  # coerce list[str]
```

Comment:
I'm confused as to why these aren't type errors anymore??

Reply:
@Kilo59 I do not know.  But Azure complained that these "ignores" are no longer needed.  I mentioned this in Slack.